qa/tasks/cephfs/test_scrub_checks.py

```diff
@@ -177,6 +177,41 @@ def promoted():
 
         self._check_task_status_na()
 
+    def test_scrub_when_mds_is_inactive(self):
+        test_dir = "scrub_control_test_path"
+        abs_test_path = f"/{test_dir}"
+
+        self.create_scrub_data(test_dir)
+
+        out_json = self.fs.run_scrub(["start", abs_test_path, "recursive"])
+        self.assertNotEqual(out_json, None)
+
+        # allow standby-replay
+        self.fs.fail()
+        self.fs.set_allow_standby_replay(True)
+        self.fs.set_joinable()
+
+        # start the scrub and verify
+        with self.assertRaises(CommandFailedError) as ce:
+            self.fs.run_scrub(["start", abs_test_path, "recursive"])
```

Comment:
I'm puzzled. Is this test not racing with the MDS becoming active?

I also expected to see a scrub command directed at the standby-replay daemon.

Reply:
Updated the PR with the proposed changes.